conjur/argparsewrapper.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
```

Comment:
is this a good filename? hard to read

Reply:
what else would you call it? I am wrapped the argparse class and override methods

Reply:
i don't know python conventions but something like `arg_parse_wrapper`.

Reply:
looked up conventions, yes `_` are allowed. updated to argparser_wrapper